src/transformers/models/auto/tokenization_auto.py

```diff
@@ -502,7 +511,7 @@ def from_pretrained(cls, pretrained_model_name_or_path, *inputs, **kwargs):
         config = kwargs.pop("config", None)
         kwargs["_from_auto"] = True
 
-        use_fast = kwargs.pop("use_fast", True)
+        use_fast = kwargs.get("use_fast", True)
```

Comment:
This means the `use_fast` stays in the kwargs, which are then sent to classes like `AutoConfig`, which don't use it. So I'd leave it as `pop` here and if it's missing in a call down there, let's add it manually.